src/executor/admin/SubmitJobExecutor.cpp

```diff
@@ -18,41 +18,12 @@ folly::Future<Status> SubmitJobExecutor::execute() {
 
     auto *sjNode = asNode<SubmitJob>(node());
     auto jobOp = sjNode->jobOp();
-    std::vector<std::string> jobArguments;
+    auto cmd = sjNode->cmd();
+    auto params = sjNode->params();
 
-    meta::cpp2::AdminCmd cmd = meta::cpp2::AdminCmd::UNKNOWN;
```

Comment:
Why move this?

Reply:
I have moved it to AdminSentence's constructor:
```C++
class AdminJobSentence final : public Sentence {
public:
    explicit AdminJobSentence(meta::cpp2::AdminJobOp op,
                              meta::cpp2::AdminCmd cmd = meta::cpp2::AdminCmd::UNKNOWN)
        : op_(op), cmd_(cmd) {
```
Because AdminCmd can be decided in parser

Reply:
Emm. I means why move this into the callback.